packages/sdks/react-sdk/src/components/UserProfile.tsx

```diff
@@ -66,10 +45,16 @@ const UserProfile = React.forwardRef<HTMLElement, UserProfileProps>(
           widgetId={widgetId}
           baseUrl={baseUrl}
           baseStaticUrl={baseStaticUrl}
```

Comment:
What about these props? don't we want to use the `.attr`?

Reply:
we can, but it is not needed, they are not referred as such (because the name is different because the camelCase vs kebab-case
![image](https://github.com/user-attachments/assets/a1f6fd33-4a7c-41a9-9ece-aa9a20d6008d)

Reply:
Might be confusing, your call